src/mctpd.c

```diff
@@ -3165,6 +3284,16 @@ static const sd_bus_vtable bus_endpoint_link_vtable[] = {
 	SD_BUS_VTABLE_END
 };
 
+static const sd_bus_vtable bus_network_vtable[] = {
+	SD_BUS_VTABLE_START(0),
+	SD_BUS_PROPERTY("LocalEids",
```

Comment:
Apologies for not picking this one up earlier, but this should be `LocalEIDs` to match the existing convention:

 * the `EID` property on `xyz.openbmc_project.MCTP.Endpoint`
 * the capitalisation of abbreviations (`MCTP`) elsewhere in property/interface names

Given that "eid" is also a word, we want to make it recognisable as the abbreviation.